### PR TITLE
Quick fixes and fixing issue for 404,500

### PR DESCRIPTION
**Pull request by shriyanka**

### CODE REVIEW

File: HTTP_200/templates/404.html [hunk 1]
```diff
@@ -1 +1,13 @@
...
+</head>
+<body>
+<div class="wrap" style="width:100%;">
+    <div class="logo" style="width:430px;position:absolute;top:25%;left:35%;">
+        <img src="{{ STATIC_URL }}images/image404.png" alt="404 error"/>
+        <p><a href="{% url 'home'%}">Go back to Home</a></p>
+    </div>
+</div>
+</body>
+</html>
```
Comment (DESHRAJ) [inline HTTP_200/templates/404.html]: Can you add a newline after the last line ?

File: HTTP_200/templates/500.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+</head>
+<body>
+<div class="wrap" style="width:100%;">
+    <div class="logo" style="width:430px;position:absolute;	top:25%;left:35%;">
+        <img src="{{ STATIC_URL }}images/image500.png" alt="500 error"/>
+        <p><a href="{% url 'home'%}">Go back to Home</a></p>
+    </div>
+</div>
+</body>
+</html>
```
Comment (DESHRAJ) [inline HTTP_200/templates/500.html]: Please add a newline after the last line.

**Comment by DESHRAJ** [comment]
@shriyanka, after doing the changes, can you please squash the commits in a single commit. 
For reference, visit http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html

File: HTTP_200/templates/500.html (hunk 2 above)
Comment (shriyanka) [inline HTTP_200/templates/500.html]: ok

**Comment by shriyanka** [comment]
Lots of conflicts will fix it tommorw

**Pull request closed without merging**